app/models/category.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class Category < ApplicationRecord
+  ##
+  # Associations
+  #
+  has_many :wissle
+  ##
+  # Validations
+  #
+  ##
+  validates :title,
+            :format => { :with => /\A[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}\Z/i },
```

Comment:
Did you copy this from `User`? Because it validates the format of a UUID (like `123e4567-e89b-12d3-a456-426655440000`). Bad title for a category methinks.